wlauto/result_processors/cpustate.py

```diff
@@ -163,8 +163,10 @@ def process_iteration_result(self, result, context):
         self.logger.debug('Generating power state reports from trace...')
         if self.create_timeline:
             timeline_csv_file = os.path.join(context.output_directory, 'power_states.csv')
+            cpu_utilisation = os.path.join(context.output_directory, 'cpu_utilisation.csv')
```

Comment:
the bool create_timeline is for creating a csv with the timeline of core power states. This is doing the same except for cpu frequency averaging. If you would like it to be its own parameter, I will first have to rename the existing one to be more explicit. Any recommendations? timeline_powerstate for example? In which case I will create a new one called timeline_cpufreq